### ISSUE TITLE
Missing AI Tools box from annotation page

### ISSUE THREAD

**Issue by Lee4396** [opencv/cvat#6864]
Hi,

I have been trying to set up my own customized server with the SAM model, however after I follow the guide I find my annotation page is missing the AI Tools like this:

![image](https://github.com/opencv/cvat/assets/43795612/28a00c2b-ab55-46af-82ab-32509c240265)

I have followed this guide with first started up cvat with the automatic annotation functions with:
docker compose -f docker-compose.yml -f components/serverless/docker-compose.serverless.yml up -d
![image](https://github.com/opencv/cvat/assets/43795612/4cab4472-5174-499a-a9df-b260941b649e)

Then I installed nuctl and ran:
./serverless/deploy_cpu.sh serverless/openvino/dextr
./serverless/deploy_cpu.sh serverless/openvino/omz/public/yolo-v3-tf
./deploy_cpu.sh pytorch/facebookresearch/sam/nuclio/

Am I missing any steps, I tried spinning up the services in different orders but it still is missing?

**Comment by stone100010** [comment]
./ deploy_gpu.sh pytorch/facebookresearch/sam/nuclio/   very well ~

**Comment by Lee4396** [comment]
> ./ deploy_gpu.sh pytorch/facebookresearch/sam/nuclio/ very well ~

Yeah but I only have cpu accessbility?

**Comment by Lee4396** [comment]
#5987 this issue is very close to this thread

**Comment by stone100010** [comment]
open your CVAT_HOST:8070
print screen.

**Comment by Lee4396** [comment]
> open your CVAT_HOST:8070 print screen.

![image](https://github.com/opencv/cvat/assets/43795612/3f388f75-36c6-48bd-8912-cb1f38d45ce6)

![image](https://github.com/opencv/cvat/assets/43795612/7925e7bc-81fb-4abb-97c7-da113b5c2652)

**Comment by bsekachev** [comment]
@Lee4396 

``docker ps``? 

Check if you didn't miss ``--build`` argument after ``docker compose ... up``.

**Comment by natanaels12** [comment]
@Lee4396 
Did you try this: `docker compose -f docker-compose.yml -f components/serverless/docker-compose.serverless.yml up -d `?

**Comment by Lee4396** [comment]
> @Lee4396
> 
> `docker ps`?
> 
> Check if you didn't miss `--build` argument after `docker compose ... up`.

Yes, I have tried with --build multiple times, I dont have access to docker ps at this moment, will do that asap

**Comment by bsekachev** [comment]
I do not have any ideas, but try to upgrade your CVAT until the latest release.

**Comment by Lee4396** [comment]
> @Lee4396
> 
> `docker ps`?
> 
> Check if you didn't miss `--build` argument after `docker compose ... up`.

I have upgrade my cvat version to v2.7.6 and tried anything again, still the same error, the model tab showed up but there is no models

I have checked someone had the same error #7037, but what I have differently is I moved everything out of the network cvat and have aws alb to do the load balancing.

The following is my docker ps
<img width="1513" alt="Screenshot 2023-11-02 at 16 10 51" src="https://github.com/opencv/cvat/assets/43795612/bb326362-8366-4652-b531-eea4b937f242">
